.github/workflows/env-setup

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-graal_ver="22.1.0"
+graal_ver="22.2.0"
```

Comment:
There is a newer version that may be more effective.

Reply:
I re-ran the trace with this version, committed the changes, and updated the version in the env setup.